### PR TITLE
fixed viz.json renderer

### PR DESCRIPTION
**Pull request by javisantana**
cc @fdansv

### CODE REVIEW

File: examples/test.html [hunk 1]
```diff
@@ -58,13 +58,18 @@
...
+      cartocss: document.getElementById('choropleth').innerHTML,
+  }];
+
   var cartoDBd3Layer = new L.CartoDBd3Layer({
     user: 'fdansv',
-    table: 'ne_adm0_europe',
-    cartocss: document.getElementById('choropleth').innerHTML,
-    tiler_template: "http://development.localhost.lan:8181"
+    layers: layers,
+    styles: layers.map(function(l) { return l.cartocss }),
```
Comment (fdansv) [inline examples/test.html:69]: wouldn't it make sense for the provider to take the style from the layer definition?

File: examples/test.html [hunk 2]
```diff
@@ -58,13 +58,18 @@
   attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
   }).addTo(map);
 
+  var layers = [{
+      sql: 'select * from ne_10m_populated_places_simple_3',
```
Comment (fdansv) [inline examples/test.html:62]: what's the expectation when we pass an sql string? we're not handling custom sqls in the provider right now

Comment (javisantana) [inline examples/test.html:62]: cartodb deals with sql not with tables, all the client side apis have been changed in order to work only with sql. Table support is only for backwards compatibility

File: examples/test.html (hunk 1 above)
Comment (javisantana) [inline examples/test.html:69]: yes, could be, but in this way rendering and layers are decoupled

File: examples/test.html (hunk 2 above)
Comment (fdansv) [inline examples/test.html:62]: Oh, sorry, didn't realise you were adding passing it when asking for the mapconfig. All good then.

**Comment by fdansv** [comment]
@javisantana :ship: :it:

**Pull request merged**